### PR TITLE
Updating README

### PR DESCRIPTION
**Pull request by machristie**
Updating README with more accurate information and removing some outdated stuff.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,135 +1,73 @@
...
+[Apache Airavata Django Portal](https://github.com/apache/airavata-django-portal).
+
+Learn more about Airavata at
+[https://airavata.apache.org](https://airavata.apache.org).
+
+## Building Apache Airavata
+
+### Prerequisites
+
+- Sources compilation require Java SDK 8 or higher.
```
Comment (DImuthuUpe) [inline README.md:24]: For master branch, we are still stuck for java 8. It doesn't work for later version

**Comment by smarru** [comment]
Thanks Marcus for tackling this overdue task, the changes look good to me.

File: README.md (hunk 1 above)
Comment (smarru) [reply README.md:24]: I guess we should tag the master branch to a release branch and merge develop to master soon.

Comment (DImuthuUpe) [reply README.md:24]: +1

Comment (machristie) [reply README.md:24]: Good catch @DImuthuUpe . I'll update the version to just 8 then when this is merged back to develop we can update it to "11 or higher".

**Pull request merged**